service/src/test/java/bio/terra/workspace/service/workspace/AzureWorkspaceTest.java

```diff
@@ -60,4 +81,64 @@ void createGetDeleteAzureContext() {
     workspaceService.deleteAzureCloudContext(workspace, userRequest);
     assertTrue(azureCloudContextService.getAzureCloudContext(workspace.getWorkspaceId()).isEmpty());
   }
+
+  @Test
```

Comment:
I started adding to this test to have it clone a storage container as well but got stuck because there was no landing zone storage account and I don't want to have to wait to spin up a landing zone or a storage account here. Let me know if you have any suggestions, as I'd like to also test that the `cloneWorkspace` flight is correctly spinning out a flight to clone the storage container